#Goal
int64 target_number
float64 period
---
#Result
int64 reached_number
---
#Feedabck
float64 current_number